src/App.tsx

```diff
@@ -23,18 +25,28 @@ import ServerStatus from './status/ServerStatus';
 
 interface State {
   requests: {
-    [key: string]: RequestStatus,
+    [key: string]: RequestObject,
   };
   isModalOpen: boolean;
-  modalMessage: string;
+  modalMessage: JSX.Element | string;
+  currentPage: number;
+  postsPerPage: number;
 }
 
+const GlobalStyle = createGlobalStyle`
```

Comment:
Please move this global styling to seperate file. Somethinbg like global.styles.ts

Reply:
done